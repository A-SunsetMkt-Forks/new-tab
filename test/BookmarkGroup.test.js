'use strict';

const BookmarkGroup = require('../src/components/BookmarkGroup.html');

describe('BookmarkGroup root component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new BookmarkGroup({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
