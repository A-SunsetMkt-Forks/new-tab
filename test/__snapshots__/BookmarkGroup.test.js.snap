// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkGroup component renders correctly 1`] = `
"<div id=\\"a\\"><div id=\\"b\\"></div>

  <a href=\\"undefined\\" title=\\"Other bookmarks\\" class=\\"b\\">
    <span class=\\"t\\">Other bookmarks</span></a><!----></div>"
`;
